play-services-fido-core/src/main/res/values-zh-rCN/strings.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?><!--
+  ~ SPDX-FileCopyrightText: 2022 microG Project Team
+  ~ SPDX-License-Identifier: Apache-2.0
+  -->
+
+<resources>
+    <string name="fido_welcome_title">在 %1$s 上使用安全密钥</string>
+    <string name="fido_welcome_body">在 %1$s 上使用安全密钥有助于保护您的隐私数据。</string>
+    <string name="fido_welcome_button_get_started">开始</string>
+    <string name="fido_welcome_privileged_info">%1$s 将充当您信任的浏览器，可将您的安全密钥用于 %2$s。</string>
+    <string name="fido_welcome_privileged_check">%1$s 是我信任的浏览器，允许其将安全密钥用于第三方网站。</string>
+    <string name="fido_transport_selection_title">选择如何使用您的安全密钥</string>
+    <string name="fido_transport_selection_body">安全密钥可通过蓝牙、NFC，或 USB 进行使用。请选择如何使用您的安全密钥。</string>
+    <string name="fido_biometric_prompt_title">验证您的身份</string>
+    <string name="fido_biometric_prompt_body">"%1$s 需要验证您的身份。"</string>
+    <string name="fido_usb_title">连接您的 USB 安全密钥</string>
+    <string name="fido_usb_prompt_body">将您的安全密钥插入到 USB 端口，或者用 USB 线缆连接。若您的密钥上有金色圆片或按钮，请点按它。</string>
+    <string name="fido_transport_selection_bluetooth">通过蓝牙使用安全密钥</string>
+    <string name="fido_transport_selection_nfc">通过 NFC 使用安全密钥</string>
+    <string name="fido_transport_selection_usb">通过 USB 使用安全密钥</string>
+    <string name="fido_transport_selection_biometric">使用此设备的屏幕锁定</string>
```

Comment:
In fact, it should refer to using this Android device as a security key, it seems that this sentence is only translated from English. @mar-v-in , what do you think?